### DISCUSSION TITLE
Javascript Webseite programmieren

### DISCUSSION THREAD

**Discussion by esthicodes** [Estheryu991/SwissGerman_Dictionary#175] [General]
Hoi zämä,
Ich sueche lüt für javascript Webseite programmieren und gleichzeitig schwiizerdütsch übersetzen, 216.66chf kann ich bezahlen für zwei Stunde, kennst du jemanden das ich öppis finden kann? Dankeschön.

website:https://github.com/esthicodes/Awesome-Swiss-German/discussions/174
 https://kdt.elice.io
Interview: asap bis 15th Feb 1am-

**Comment by esthicodes** [comment]
Erstellen und rufen Sie eine Funktion auf

Lassen Sie uns eine Funktion vervollständigen, die Eingabeparameter gemäß den Anweisungen hinzufügt, und das Ergebnis des Aufrufs der Funktion drucken.
Anweisung

Erstellen Sie eine Funktion namens add in der Lösungsfunktion, die das Ergebnis der Addition der beiden Eingabeparameter mit zwei Parametern x und y zurückgibt
Geben Sie diese Funktion aus der Lösungsfunktion zurück.
Tipps

Wenn Sie eine Funktion innerhalb einer Funktion zurückgeben, schreiben Sie nur den Funktionsnamen, ohne die Funktion aufzurufen.
Wenn beispielsweise add() aufgeschrieben wird, wird das Rückgabeergebnis der add-Funktion an die Lösungsfunktion zurückgegeben. Daher muss es als add geschrieben werden.

**Reply by esthicodes** [reply]
![BEAD924E-B572-4055-B524-761CB090B415](https://user-images.githubusercontent.com/78131082/218719287-ed2e056e-8893-4b41-8226-b4c157f84db3.jpeg)

Sie können eine Zeichenfolge in der Lösungsfunktion zurückgeben. Üben Sie Fragen, um das Ellis Scoring Tool zu verstehen.

Funktion Lösung() {

   geben Sie "Hallo, Alice" zurück;
}
Sie können auch console.log() überall dort verwenden, wo Sie sehen müssen, wie Ihr Code funktioniert.

**Reply by ThomasG777** [reply]
Erstelle und ruefed sie e Funktion uf

lönd sie üs e Funktion vervollständige, wo d Igabparameter gemäss de awisige hinzuefügt, 
und s ergebnis vom Ufruef vo de funktion drucke

Awisig

Erstelled sie e funktion namens add i de lösigsfunktion, 
wos Ergebnis vo de Addition vo de beide Igabeparameter mit zwei Parameter x und y zrugggit
gäbet sie die funktion us de lösigsfunktion zrugg.

tipps

Wenn sie e Funktion innerhalb enere Funktion zrugggäbet,  schribet sie nur Funktionsname, ohni d funktion ufzruefe.
Wenn bispielswis add() ufgschribe wird, wird s Rückgabeergebnis vo de add-funktion ad Lösigsfunktion zrugggäh.
darum muess es als add gschribe werde.

**Reply by ThomasG777** [reply]
Sie chönd e zeichefolg i de Lösigsfunktion zrugggäh. übet sie frage, ums Ellis Scorin Tool zverstah.
Funktion Lösig() 
gebte Sie "Hallo, Alice" zrugg

Sie chönd au console.log() überall det verwende, wo sie gseh müend, wie ihre Code funktioniert.

**Comment by esthicodes** [comment]
![95FACCA4-6ACD-4CF8-95F0-70075C203CDA](https://user-images.githubusercontent.com/78131082/218720196-b8c5e84d-b964-4267-ba6d-0abdad764dc8.jpeg)

**Comment by esthicodes** [comment]
![8CCF5220-76AA-41DE-8A0E-A82D28205C75](https://user-images.githubusercontent.com/78131082/218766755-3ad28f8e-2660-4beb-9115-22c23be3a5c0.png)
![18F85474-C583-4120-828D-9A2AB8180F20](https://user-images.githubusercontent.com/78131082/218766796-b42889b4-26e3-4e2c-b479-4aad12436938.png)
![C3C0DE3E-A834-447E-AA15-358CD85CD598](https://user-images.githubusercontent.com/78131082/218766807-b3bb6498-9b4f-437c-af67-7a12c1e979ab.png)
![4E458D25-8380-49DC-9B41-10EFC594AC8A](https://user-images.githubusercontent.com/78131082/218766808-b2dca2c3-bdcc-4b76-b97a-6a071f0bd142.png)
![FC8FCF3F-7777-47F2-925C-635B51EEB7B3](https://user-images.githubusercontent.com/78131082/218766812-92efc707-c623-482a-8a86-155ac67cd0d1.png)
![58C40378-A9F8-4333-A232-0DBD4FF7BE71](https://user-images.githubusercontent.com/78131082/218766814-e0b81c61-9841-47b8-9c42-57df5d934589.png)
![F4483C8F-7085-46D3-9C0B-464777F2588A](https://user-images.githubusercontent.com/78131082/218766817-7afdee30-b373-4af8-9dba-a41aa79f8a4f.png)
![1B9CAC96-1E96-4578-B3A9-DC92C7066423](https://user-images.githubusercontent.com/78131082/218766823-379b0c6a-bae6-4521-b599-416ff101b9b6.png)
![CFA67546-7699-4A93-A67B-8086A8043CEE](https://user-images.githubusercontent.com/78131082/218766825-b8c223fa-7a89-4d26-b86d-dc8fe6084193.png)
![EF472E67-64ED-412B-8466-0CBC05D77B80](https://user-images.githubusercontent.com/78131082/218766828-765b3d29-8b8e-4170-bc30-376fcfaa6790.png)
